src/unlimited.cpp

```diff
@@ -1814,8 +1829,22 @@ UniValue getstat(const UniValue &params, bool fHelp)
         }
         else
         {
+            //Get time before request
+            int64_t SeriesStartTime = GetTimeMillis();
```

Comment:
Ok this is confusing. My intent was to indicate the points before and after the call to the stat class so that if the presented series crossed over when a series would have been calculated the user would have a way to know that. 

If we move the time calc into the stat class this issue is mitigated.